crepecake/src/main/java/com/google/cloud/tools/crepecake/image/json/V21ManifestTemplate.java

```diff
@@ -55,6 +55,8 @@
  */
 public class V21ManifestTemplate extends ManifestTemplate {
 
+  public static final String MEDIA_TYPE = "application/vnd.docker.distribution.manifest.v1+json";
```

Comment:
do docker versioning names match up funny? or is 2.1 just compatible with all v1 images?

Reply:
Yea, Docker's versioning only applies to Version 2 images. The v1 (or v2 in V22ManifestTemplate) refers to the Schema version, such that [Version 2 Schema 1 is v1](https://docs.docker.com/registry/spec/manifest-v2-1/) and Version 2 Schema 2 is v2.